shopify/slack-notification-template/api/models/shopifyShop/actions/uninstall.ts

```diff
@@ -25,7 +21,7 @@ export async function run({ params, record, logger, api }) {
 
   if (record.slackChannelId) {
     await slackClient.conversations.leave({
-      token: record.slackAccessToken,
+      token: record.slackAccessToken as string,
```

Comment:
same here, throw if the slackAccessToken is null or undef

Reply:
I changed the as string to a check in the if